templates/cluster.yml.j2

```diff
@@ -15,6 +15,7 @@ spec:
   docker:
     insecureRegistry: 100.64.0.0/10
     logDriver: ""
+  kubeAPIServer: "{{ cluster.kube_api_server }}"
```

Comment:
As you can see here: https://github.com/Flaconi/ansible-role-kops/pull/13/files#diff-04c6e90faac2675aa89e2176d2eec7d8L104

we only have three **required** keys:

* `name`
* `s3_bucket_name`
* `workers`

That means the `kubeAPIServer` directive can try to access a value that is not set and fail.
To circumvent this, you should wrap this into an if condition:
```jinja2
{% if 'cluster.kube_api_server'  in cluster %}
kubeAPIServer: "{{ cluster.kube_api_server }}"
{% endif %}
```